static/app/components/notificationActions/notificationActionManager.tsx

```diff
@@ -207,12 +205,11 @@ function NotificationActionManager({
           // Add notification action
           const updatedActions = [...notificationActions, validActions[0].action];
           setNotificationActions(updatedActions);
-          updateAlertCount(parseInt(project.id, 10), updatedActions.length);
         },
       });
     });
     return dropdownMenuItems;
-  }, [actionsMap, availableServices, notificationActions, project, updateAlertCount]);
+  }, [actionsMap, availableServices, notificationActions]);
```

Comment:
Was it intended to remove `project` here too?

Reply:
yes, because `project` is only used inside `updateAlertCount`, and we don't have a dependency on it anymore

Reply:
There's no dependency on `project` in the `useMemo` callback.